src/components/Onfido/index.native.js

```diff
@@ -45,6 +46,29 @@ class Onfido extends React.Component {
                     return;
                 }
 
+                // Handle user camera permission on iOS and Android
+                if (_.contains([CONST.ONFIDO.ERROR.USER_CAMERA_PERMISSION, CONST.ONFIDO.ERROR.USER_CAMERA_DENINED, CONST.ONFIDO.ERROR.USER_CAMERA_CONSENT_DENIED], errorMessage)) {
```

Comment:
Please wrap this next lines. Something like this
```suggestion
                if (_.contains([
                	CONST.ONFIDO.ERROR.USER_CAMERA_PERMISSION,
                	CONST.ONFIDO.ERROR.USER_CAMERA_DENINED,
                	CONST.ONFIDO.ERROR.USER_CAMERA_CONSENT_DENIED
                	], errorMessage)
                ) {
```